### PR TITLE
Add a LICENSE file and do some cleanup

### PR DESCRIPTION
**Pull request by HavokInspiration**

### CODE REVIEW

**Comment by dereuromark** [comment]
:+1:

File: src/Util/ColumnParser.php [hunk 1]
```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```
Comment (josegonzalez) [inline src/Util/ColumnParser.php]: I had this originally to avoid some lint failures in my editor *shrug*.

Comment (HavokInspiration) [inline src/Util/ColumnParser.php]: To be honest, I find it weird to have a variable dropped just like that.

(What does "my editor shrug" means ?).

Comment (josegonzalez) [inline src/Util/ColumnParser.php]: Atom has integration with all manner of linting programs, and in this case it was complaining about unused variables.

Comment (dereuromark) [inline src/Util/ColumnParser.php]: I also don't like code be added just for linters to be happy.
If anything it should be commented why it is done.

Comment (josegonzalez) [inline src/Util/ColumnParser.php]: Yeah just stating why I initially added that in case anyone was confused. I don't care either way :)

Comment (HavokInspiration) [inline src/Util/ColumnParser.php]: Mine does too complain about an unused variable, but it also complained about the previous state.
It gave me a "Expression result unused".
So what do we do ?

Comment (HavokInspiration) [inline src/Util/ColumnParser.php]: @josegonzalez I understand then, I found another one in another method that I removed in a previous PR.

Comment (josegonzalez) [inline src/Util/ColumnParser.php]: Whatever you want as the master of the migrations plugin :)

Comment (dereuromark) [inline src/Util/ColumnParser.php]: In what PHP version will `function (, $constant)` be allowed? :)

**Pull request merged**